clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceServerConfig.java

```diff
@@ -508,7 +508,7 @@ public VeniceServerConfig(VeniceProperties serverProperties, Map<String, Map<Str
     serverCalculateQuotaUsageBasedOnPartitionsAssignmentEnabled =
         serverProperties.getBoolean(SEVER_CALCULATE_QUOTA_USAGE_BASED_ON_PARTITIONS_ASSIGNMENT_ENABLED, true);
 
-    nodeCapacityInRcu = serverProperties.getLong(SERVER_NODE_CAPACITY_RCU, 50000);
+    nodeCapacityInRcu = serverProperties.getLong(SERVER_NODE_CAPACITY_RCU, 100000);
```

Comment:
Is the default value change intentional?

Reply:
Yes, 50k is too low for server node level qps as default IMO.